### PR TITLE
mobile: Revert accidental test changes

### PR DESCRIPTION
**Pull request by fredyw**
Some `CronetHttp3Test` tests got accidentally commented out. This PR revert these accidental changes.

Risk Level: low
Testing: `blaze test //test/java/org/chromium/net:cronet_http3_test`
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: n/a

### CODE REVIEW

File: mobile/test/java/org/chromium/net/CronetHttp3Test.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public class CronetHttp3Test {
 
   // If true, dump envoy logs on test completion.
   // Ideally we could override this from the command line but that's TBD.
-  private boolean printEnvoyLogs = true;
+  private boolean printEnvoyLogs = false;
```
Comment (abeyad) [inline mobile/test/java/org/chromium/net/CronetHttp3Test.java:43]: should we keep this as true?  then we'd get more useful info on test failures

Comment (fredyw) [reply mobile/test/java/org/chromium/net/CronetHttp3Test.java:43]: I think initially this was set to false, but I agree that true is actually better. Let me fix it.

Comment (alyssawilk) [reply mobile/test/java/org/chromium/net/CronetHttp3Test.java:43]: let's keep off by default and enable if tests get flaky

**Review by alyssawilk — DISMISSED** [review]

File: mobile/test/java/org/chromium/net/CronetHttp3Test.java (hunk 1 above)
Comment (fredyw) [reply mobile/test/java/org/chromium/net/CronetHttp3Test.java:43]: Done. Setting it back to `false`.

Comment (alyssawilk) [reply mobile/test/java/org/chromium/net/CronetHttp3Test.java:43]: haha, either way really

**Review by abeyad — APPROVED** [review]

**Pull request merged**